apps/watch/src/components/Video/VideoContent/VideoContent.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { Meta, Story } from '@storybook/react'
+import { ComponentProps } from 'react'
+import { VideoProvider } from '../../../libs/videoContext'
+import { watchConfig } from '../../../libs/storybook'
+import { videos } from '../../Videos/testData'
+import { VideoContent } from './VideoContent'
+
+const VideoContentStory = {
+  ...watchConfig,
+  component: VideoContent,
+  title: 'Watch/Video/VideoContent'
+}
+
+const Template: Story<ComponentProps<typeof VideoContent>> = () => (
```

Comment:
```suggestion
const Template: Story = () => (
```

No need for component props as we're not using any of VideoContents props anymore